label_namer.go

```diff
@@ -57,10 +57,6 @@ func (ln *LabelNamer) Build(label string) (normalizedName string, err error) {
 			return
 		}
 
-		if ln.UTF8Allowed || normalizedName == label {
```

Comment:
I think this function can now be rewritten without a `defer`, and just do the error checking at the end of the function as normal. There used to be a bunch of return cases but those have been disappearing over time, so I think the defer now just makes it overly complicated

Reply:
fair, removed the defer